Formula/filebeat.rb

```diff
@@ -0,0 +1,77 @@
+class Filebeat < Formula
+  desc "File harvester, used to fetch logs files and feed them into logstash"
+  homepage "https://www.elastic.co/products/beats/filebeat"
+  url "https://github.com/elastic/beats.git",
+    :tag => "v1.2.3",
+    :revision => "e849e45f713a6112145ac8309fc23bd69bc41ef8"
+
+  head "https://github.com/elastic/beats.git"
+
+  depends_on "go" => :build
+
+  def install
+    contents = Dir["{*,.git,.gitignore}"]
+    gopath = buildpath/"gopath"
+    (gopath/"src/github.com/elastic/beats").install contents
+
+    ENV["GOPATH"] = gopath
+
+    cd gopath/"src/github.com/elastic/beats/filebeat" do
+      system "go", "get"
+      system "make"
+      libexec.install "filebeat"
+      etc.install "etc/filebeat.yml"
+    end
+
+    sh = bin/"filebeat"
+    sh.write "#!/usr/bin/env bash\n\n#{libexec}/filebeat -c #{etc}/filebeat.yml \"$@\""
```

Comment:
Will do the changes for `EOS.undent`. 

Regarding the user specifying the configuration file it would work as the `-c` option can be passed multiple times and the last one is going to be the one actually used (as you can see in the test). But you may have a point. I did that because, when compiling, the generated binary expects the default configuration to be in `$HOME/filebeat.yml`. I though it would be better to have it in `$HOMEBREW/etc/filebeat.yml` (because it's more the "unix way"). So what's your opinion on this? Let the default behaviour alone and remove this wrapper? Or maybe try to see if I can change the default path during compilation to avoid using this wrapper? Or simply keep it this way?